### DISCUSSION TITLE
No more account login on devices, instead one-time tokens

### DISCUSSION THREAD

**Discussion by serrq** [mullvad/mullvadvpn-app#8966] [Feature requests & Ideas]
### I have checked if others have suggested this already

- [X] I have checked this issue tracker to see if others have reported similar issues.

### Feature description

My idea is to remove entirely the 16 digits account from devices.
Lost and shared devices show the 16 digits account, this is a security issue for a user.

### Alternative solutions

The main idea is using the web dashboard to make login with the 16 digits and once within the dashboard generating tokens for each devices, directly from the website.

The new login on web dashboard could be made with 24 words passphrase. Better security while keeping simplicity.

### Type of feature

- [X] Better privacy/anonymity
- [ ] Better at circumventing censorship
- [ ] Easier to use
- [ ] Other

### Operating System

- [X] Android
- [X] iOS
- [X] Windows
- [X] macOS
- [X] Linux

**Comment by MrChocolatine** [comment]
> The main idea is using the web dashboard to make login with the 16 digits and once within the dashboard generating tokens for each devices, directly from the website.

Wouldn't that make the user experience quite tedious? Every time, having to connect on the website and then generate a token.
In the end, as you explain, the 16 digits token would still exist since it would still be required to log in to the website.
